### PR TITLE
Add test profiles blog

### PR DESCRIPTION
**Pull request by stuartwdouglas**
** If you are updating a guide, please submit your pull request to the main repository: https://github.com/quarkusio/quarkus/tree/master/docs/src/main/asciidoc **

### CODE REVIEW

File: _posts/2020-07-03-quarkus-test-profiles.adoc [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+---
+
+= Quarkus now supports test profiles
+
+With the release of Quarkus 1.6 Quarkus now supports test profiles, allowing you to easily test multiple different configurations
+inside the same module. This post outlines what test profiles are and how you can use them.
+
+== Recap on how `@QuarkusTest` works
+
+Before we talk about test profiles lets talk briefly about how `@QuarkusTest` annotated tests actually work. The first time
```
Comment (geoand) [inline _posts/2020-07-03-quarkus-test-profiles.adoc:16]:
```suggestion
Before we talk about test profiles let's talk briefly about how `@QuarkusTest` annotated tests actually work. The first time
```

File: _posts/2020-07-03-quarkus-test-profiles.adoc [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+one of these tests is run the Quarkus test extension will start Quarkus. Quarkus will then remain running for the rest
```
Comment (geoand) [inline _posts/2020-07-03-quarkus-test-profiles.adoc:17]:
```suggestion
one of these tests is run the Quarkus test extension will start Quarkus. Quarkus will then remain running for the duration
```

File: _posts/2020-07-03-quarkus-test-profiles.adoc [hunk 3]
```diff
@@ -0,0 +1,105 @@
...
+of the test run. This makes testing very fast, because Quarkus is only started once, however it does limit you to testing
+a single configuration per module, as you can't restart Quarkus in a different configuration. Test profiles remove this
```
Comment (geoand) [inline _posts/2020-07-03-quarkus-test-profiles.adoc:19]:
```suggestion
a single configuration per module, as you can't restart Quarkus in a different configuration. Test profiles lift this
```

File: _posts/2020-07-03-quarkus-test-profiles.adoc [hunk 4]
```diff
@@ -0,0 +1,105 @@
...
+limitation.
+
+== Test profiles
+
+Test profiles are defined by the `io.quarkus.test.junit.QuarkusTestProfile` interface. To define an profile you need to
```
Comment (geoand) [inline _posts/2020-07-03-quarkus-test-profiles.adoc:24]:
```suggestion
Test profiles are defined by the `io.quarkus.test.junit.QuarkusTestProfile` interface. To define a profile you need to
```

File: _posts/2020-07-03-quarkus-test-profiles.adoc [hunk 5]
```diff
@@ -0,0 +1,105 @@
...
+public class MyTestClass {
+...
+}
+----
+
+This test class will now be run with the settings specified in the `MyProfile` class.
+
+== How it works
+
+Before a `@QuarkusTest` is executed the Quarkus test extension will check the profile that the test wants to use, and compares
```
Comment (geoand) [inline _posts/2020-07-03-quarkus-test-profiles.adoc:92]:
```suggestion
Before a `@QuarkusTest` is executed, the Quarkus test extension will check the profile that the test wants to use, and compare
```

Comment (stuartwdouglas) [reply _posts/2020-07-03-quarkus-test-profiles.adoc:92]: compares is correct here.

Comment (geoand) [reply _posts/2020-07-03-quarkus-test-profiles.adoc:92]: I beg to differ, but it's too minor 😎

**Pull request merged**